xbmc/cores/VideoPlayer/DVDInputStreams/InputStreamAddon.cpp

```diff
@@ -137,6 +138,16 @@ bool CInputStreamAddon::Open()
     props.m_ListItemProperties[props.m_nCountInfoValues].m_strKey = pair.first.c_str();
     props.m_ListItemProperties[props.m_nCountInfoValues].m_strValue = pair.second.c_str();
     props.m_nCountInfoValues++;
+
+    if (props.m_nCountInfoValues >= STREAM_MAX_PROPERTY_COUNT)
+    {
+      CLog::Log(LOGERROR, 
+                "CInputStreamAddon::Open - Hit max count of stream properties, "
```

Comment:
About here, maybe use a `CInputStreamAddon::%s` and `__func__` (new C++11) to prevent name mismatch if this is moved somewhere else.